src/components/Root/Root.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    minWidth: theme.breakpoints.values['xs'],
+  },
+}));
+
+const Root: React.FunctionComponent = () => {
+  const classes = useStyles();
+
+  return <Container className={classes.container} component='main' maxWidth='md'></Container>;
```

Comment:
It throws error when `<Container>` tag i empty. At least in my local env. Could you confirm that? Maybe it's sth wrong with my setup....

![image](https://user-images.githubusercontent.com/14184100/66372949-a6471f80-e9a7-11e9-8296-277e0ef89dbf.png)

Reply:
Added a dummy Text to avoid this error for now